### PR TITLE
Battalion

### PR DESCRIPTION
**Pull request by MadShmupper**
Add New Port Battalion

# New Port for { Battalion }

## Game Information
- **Title**: { Battalion }
- **URL**: { https://all-x.itch.io/battalion }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/battalion/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./BATTALION.sh</path>
```
Comment (Slayer366) [inline ports/battalion/gameinfo.xml:4]:
```suggestion
    <path>./Battalion.sh</path>
```

File: ports/battalion/port.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+    "version": 3,
+    "name": "battalion.zip",
+    "items": [
+        "BATTALION.sh",
```
Comment (Slayer366) [inline ports/battalion/port.json:5]:
```suggestion
        "Battalion.sh",
```
Your shell script is not in all caps.  You can change the shell script to be in all caps or you'll need to change the entries in your gameinfo.xml and port.json as per my suggestions to match the shell script name especially with case sensitivity.

File: ports/battalion/port.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        "image": null,
+        "rtr": true,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "armhf"
+        ]
+    },
+    "status": {
```
Comment (Slayer366) [inline ports/battalion/port.json:31]: You have these entries created by harbourmaster in your port.json
```suggestion
    "status": {
```

File: ports/battalion/port.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        "source": "Unknown",
+        "md5": null,
+        "status": "Installed"
+    },
+    "files": {
```
Comment (Slayer366) [inline ports/battalion/port.json:36]: You have these entries created by harbourmaster in your port.json
```suggestion
    "files": {
```

**Review by Slayer366 — CHANGES_REQUESTED** [review]

**Review by kloptops — APPROVED** [review]
👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**